### PR TITLE
feat(protocol-designer): add 3.20 export warning modal

### LINKED ISSUE #6278: feat: PD v5 export warning modal
# Overview

As a user I would like to be informed that I need to update my robot to `3.20.0` in order to use protocols with air gap or delay functionality.

# Implementation details
- Add v5 export check logic and modal text to existing export blocking hints

no modules, no airGap, no moveToWell: use v3
has modules: at least v4
has airGap or moveToWell: at least v5

# Design
https://www.figma.com/file/0dTEbGNsKYgNwKEW6LLfzJ/Airgap-Delay?node-id=395%3A36

### PR DESCRIPTION
**Pull request by shlokamin**
# Overview

This PR closes #6278 by adding a 3.20 export warning modal if the user attempts to export a protocol with an air gap or a delay.

# Changelog

- Add 3.20 export warning modal 

# Review requests
- Export protocol with no modules, no air gap/delays, export modal should not show up
- Export protocol with modules, no air gap/delays, export 3.18 export modal should show up
- Export protocol with air gap/delays, export 3.20 export modal should show up

# Risk assessment
Low

### CODE REVIEW

File: protocol-designer/src/components/FileSidebar/FileSidebar.js [hunk 1]
```diff
@@ -199,13 +210,29 @@ export function FileSidebar(props: Props): React.Node {
       modulesWithoutStep,
     })
 
-  const blockingV4ExportHint = useBlockingHint({
-    enabled: isV4Protocol && showV4ExportWarning,
-    hintKey: 'export_v4_protocol_3_18',
-    content: v4WarningContent,
-    handleCancel: () => setShowV4ExportWarning(false),
+  // type T = {| isV4: boolean |}
```
Comment (shlokamin) [inline protocol-designer/src/components/FileSidebar/FileSidebar.js:213]: oop this is left over

Comment (IanLondon) [inline protocol-designer/src/components/FileSidebar/FileSidebar.js:213]: commented-out code

**Comment by IanLondon** [comment]
Modal doesn't match copy in Figma, repeats itself image

**Review by IanLondon — APPROVED** [review]
approved pending modal copy fixup, code looks good

**Comment by shlokamin** [comment]
Copy text fixed!

**Pull request merged**